### ISSUE TITLE
Virtual text not visible when 'signcolumn' is set to 'yes'

### ISSUE THREAD

**Issue by bfrg** [vim/vim#10794]
### Steps to reproduce

Run `vim --clean test.vim -S test.vim`, with
```vim
vim9script
&signcolumn = 'yes'
prop_type_add('test', {highlight: 'Special'})
prop_add(1, col('$'), {
    type: 'test',
    text: ' the quick brown fox jumps over the lazy dog'
})
```

Result: The virtual text is not visible.

### Expected behaviour

Virtual text should always be visible.

### Version of Vim

9.0.72

### Environment

Linux

**Comment by Shane-XB-Qian** [comment]
BTW: maybe due to difficult to calc `col('$')` ?

**Comment by puremourning** [comment]
no, this is reproducable without that:

```
" This line

set signcolumn=yes
silent! call prop_type_delete( "a_test" )
call prop_type_add( "a_test", #{ highlight: 'Error' } )
call prop_add( 1, 12, #{ text: 'Some text', type: 'a_test' } )

```

**Comment by brammool** [comment]
Fixed with patch 9.0.0116

**Comment by brammool** [comment]
> BTW: maybe due to difficult to calc `col('$')` ?

That should not be the problem.  Behavior at the end of the line needs
to be figured out (adding virtual text after the buffer line which
sticks there even when the buffer line gets longer).

I first want to work away various other issues and PRs before I continue
with this.

-- 
We are the Borg of GNU GPL.  We will assimilate your source code.
Resistance is futile.

 /// Bram Moolenaar -- ***@***.*** -- http://www.Moolenaar.net   \\\
///                                                                      \\\
\\\        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
